### DISCUSSION TITLE
Time series to supervised

### DISCUSSION THREAD

**Discussion by PARODBE** [alan-turing-institute/sktime#3704] [Q&A]
Hi,

First of all, thank you for this so useful library. I have a question since I just came across with your library. If you want convert the typical time series problem to supervised (input and output data are continuos), something like this: https://machinelearningmastery.com/convert-time-series-supervised-learning-problem-python/

Do you have any function or proposal for this?

Thanks!
Pablo

**Comment by fkiraly** [comment]
(moved to the discussion forum where Q&A and general advice lives)

Can you kindly describe what does your data look like, and what problem/task you want to solve?

**Reply by PARODBE** [reply]
Of course, I have the typical problem for prediction of an endemic disease. This dataset only contains numeric input data: basically variables based on weather and the output is also numeric. I'd like to convert time series problem to supervised.
